install/kubernetes/helm/istio/charts/tracing/values.yaml

```diff
@@ -69,7 +69,7 @@ service:
   annotations: {}
   name: http
   type: ClusterIP
-  externalPort: 9411
+  externalPort: 80
```

Comment:
Problem is that this property (`.Values.service.externalPort`) is also referenced in the `zipkin` [service](https://github.com/istio/istio/blob/master/install/kubernetes/helm/istio/charts/tracing/templates/service.yaml#L24), so would mess up trace collection.

Reply:
I am aware.  See followup commit, please.